concepticondata/conceptlists/Koelle-1854-335.tsv

```diff
@@ -0,0 +1,336 @@
+Koelle-1854-335-7	7	Arrow	977	ARROW
+Koelle-1854-335-8	8	a walled town		
+Koelle-1854-335-9	9	Axe	677	AXE
+Koelle-1854-335-10	10	Bad	1292	BAD
+Koelle-1854-335-11	11	Bat	1793	BAT
+Koelle-1854-335-12	12	Bat (large)		
+Koelle-1854-335-13	13	Beans	832	BEAN
+Koelle-1854-335-14	14	Bed	1663	BED
+Koelle-1854-335-15	15	Bee	665	BEE
+Koelle-1854-335-16	16	Belly	1251	BELLY
+Koelle-1854-335-17	17	Bench, Chair		
+Koelle-1854-335-18	18	Bench, Chair (for men)		
+Koelle-1854-335-19	19	Bench, Chair (for women)		
+Koelle-1854-335-20	20	Bird	937	BIRD
+Koelle-1854-335-21	21	Black	163	BLACK
+Koelle-1854-335-22	22	Black Man (Negro)	2806	BLACK MAN
+Koelle-1854-335-23	23	Blood	946	BLOOD
+Koelle-1854-335-24	24	body	1480	BODY
+Koelle-1854-335-25	25	Bone	1394	BONE
+Koelle-1854-335-26	26	Book	963	BOOK
+Koelle-1854-335-27	27	Bow	994	BOW
+Koelle-1854-335-28	28	Boy	1366	BOY
+Koelle-1854-335-29	29	Buck		
+Koelle-1854-335-30	30	Bull	1008	BULL
+Koelle-1854-335-31	31	Butter	1245	BUTTER
+Koelle-1854-335-32	32	Butterfly	1791	BUTTERFLY
+Koelle-1854-335-33	33	Butter (melted)		
+Koelle-1854-335-34	34	Calabash	411	GOURD
+Koelle-1854-335-35	35	Camwood		
+Koelle-1854-335-36	36	Canoe	1970	CANOE
+Koelle-1854-335-37	37	Cap	1288	CAP
+Koelle-1854-335-38	38	Cassada	925	CASSAVA
+Koelle-1854-335-39	39	Cassada (wild)	925	CASSAVA
+Koelle-1854-335-40	40	Cat	1208	CAT
+Koelle-1854-335-41	41	Chain (Fetters ?)		
+Koelle-1854-335-42	42	Chain (Fetters ?) (for neck)		
+Koelle-1854-335-43	43	Chamelion	1555	CHAMELEON
+Koelle-1854-335-44	44	Chest	1592	CHEST
+Koelle-1854-335-45	45	Coal	2658	COAL
+Koelle-1854-335-46	46	Cock	1511	ROOSTER
+Koelle-1854-335-47	47	Cold	1287	COLD
+Koelle-1854-335-48	48	Cotton	1850	COTTON
+Koelle-1854-335-49	49	Cotton-plant (a Shrub)		
+Koelle-1854-335-50	50	Cotton-tree		
+Koelle-1854-335-51	51	cough		
+Koelle-1854-335-52	52	Cow	1007	COW
+Koelle-1854-335-53	53	Crooked (bent)	297	CROOKED
+Koelle-1854-335-54	54	cross-bow	3193	CROSSBOW
+Koelle-1854-335-55	55	Daughter (My, Thy)	1357	DAUGHTER
+Koelle-1854-335-56	56	Day	1225	DAY (NOT NIGHT)
+Koelle-1854-335-57	57	Devil	1973	DEMON
+Koelle-1854-335-58	58	Dew	1977	DEW
+Koelle-1854-335-59	59	Doctor	597	PHYSICIAN
+Koelle-1854-335-60	60	Dog	2009	DOG
+Koelle-1854-335-61	61	Door	1567	DOOR
+Koelle-1854-335-62	62	Door (of grass)		
+Koelle-1854-335-63	63	Door (of wood)		
+Koelle-1854-335-64	64	Door-way		
+Koelle-1854-335-65	65	Drum	908	DRUM
+Koelle-1854-335-66	66	Dry	1398	DRY
+Koelle-1854-335-67	67	Dry Season	452	DRY SEASON
+Koelle-1854-335-68	68	Ear	1247	EAR
+Koelle-1854-335-69	69	Ear-Ring	770	EARRING
+Koelle-1854-335-70	70	Egg	744	EGG
+Koelle-1854-335-71	71	Eight	1705	EIGHT
+Koelle-1854-335-72	72	Eighteen	1713	EIGHTEEN
+Koelle-1854-335-73	73	Elbow	981	ELBOW
+Koelle-1854-335-74	74	Elder Brother (My, Thy)	1759	OLDER BROTHER
+Koelle-1854-335-75	75	Elder Sister	1758	OLDER SISTER
+Koelle-1854-335-76	76	Elephant	1290	ELEPHANT
+Koelle-1854-335-77	77	Eleven	1706	ELEVEN
+Koelle-1854-335-78	78	Ewe	1345	EWE
+Koelle-1854-335-79	79	Ewe (Sheep)	1345	EWE
```

Comment:
EWE is mapped twice.

Reply:
I decided to do so (as in the case of FIREWOOD), because for some languages the gloss is `Ewe` and for others it is `Ewe (Sheep)`. No language has both and exploring the data (particularly for West African languages) indicated that they are cognates.

We could keep only the item 78, far more frequent, if it is not allowed to have two glosses mapped to the same concept set.

Reply:
I agree that it then makes sense to map it to both. But I guess if we switch to the origin list anyway this problem doesn't apply anymore.

Reply:
They are actually in the source. My guess is that it indicates that the language has no different word for EWE and SHEEP (only a different one for RAM, or perhaps for the castrated one -- how is it called in English?), so that in theory a concept EWE OR SHEEP.

Reply:
Found it, castrated male sheep is "wether". Luckily it is not in this list.